docs/tsc/meetings/2019-02-04.md

```diff
@@ -31,17 +31,18 @@ Attendees:
 * John Mertic requested candidates provide their background for inclusion in the voting process with the aim of opening voting on the 5th February.
 * Secretary - in view of no volunteers, suggested we might consider rotating role.
 * Infrastructure
-  * Git needs migrating to new organisation
-  * Must perform a license/code audit prior to migration
+  * GitHub repo needs migrating to the ASWF organisation
+  * **TODO** - John Mertic to coordinate with Steve Winslow at the LF to complete.
   * https://github.com/AcademySoftwareFoundation/tac/blob/master/process/contributing.md
-* Require that any existing copyright notices are amended by the current stated owners.
-  * Once migrated will enable sign off checking, etc
+* Require that any existing copyright notices are reviewed and amended by the current stated owners if their respective legal agrees.
+  * Once migrated will enable DCO sign off checking and CLA checks.
 * Sub-discussion on the option of separate sign off for committer vs developer sign off.
 * **TODO** - John Mertic to determine if sign off lines need to be in the contribution license agreement list
 * CLAs from corporations and individuals would preferentially be in place prior to migration, John Mertic to action the set-up.
 * Need to transfer DNS and website registration 
 * **TODO** - Larry Gritz to action transfer
 * Discussion around transfer of mailing lists/slack channel
+* **TODO** - John Mertic to confirm if Google Groups can be migrated to groups.io, and pricing/options for Slack to Rocket.chat or Mattermost migration
```

Comment:
'to confirm if Google Groups can be migrated' --> 'to confirm if Google Groups and historic can be migrated'

The existing groups are: ocio-dev, ocio-users and ocio-v2

Reply:
Good catch - fixed now.